{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:ubertidavide:fastbots:*:*:*:*:*:*:*:*"
        ],
        "fix_commit": "https://github.com/ubertidavide/fastbots/commit/73eb03bd75365e112b39877e26ef52853f5e9f57",
        "version_end_excluding": {
            "ubertidavide:fastbots": "0.1.5"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "Apache License 2.0"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2023-48699"
        ],
        "CWE": [
            "CWE-95"
        ]
    }
}